data/sql/updates/characters/2016_MM_DD_XX.sql

```diff
@@ -0,0 +1,49 @@
+-- `character_aura`
+ALTER TABLE `character_aura`
+CHANGE `caster_guid`      `casterGuid`      bigint(20) unsigned NOT NULL DEFAULT '0' COMMENT 'Full Global Unique Identifier',
+CHANGE `item_guid`        `itemGuid`        bigint(20) unsigned NOT NULL DEFAULT '0',
+CHANGE `effect_mask`      `effectMask`      tinyint(3) unsigned NOT NULL DEFAULT '0',
+CHANGE `recalculate_mask` `recalculateMask` tinyint(3) unsigned NOT NULL DEFAULT '0',
+CHANGE `stackcount`       `stackCount`      tinyint(3) unsigned NOT NULL DEFAULT '1',
+CHANGE `maxduration`      `maxDuration`     int(11) NOT NULL DEFAULT '0',
+CHANGE `remaintime`       `remainTime`      int(11) NOT NULL DEFAULT '0',
+CHANGE `remaincharges`    `remainCharges`   tinyint(3) unsigned NOT NULL DEFAULT '0';
+
+-- `character_glyphs`
+ALTER TABLE `character_glyphs`
+CHANGE `spec` `talentGroup` tinyint(3) unsigned NOT NULL DEFAULT '0';
+
+-- `character_talent`
+ALTER TABLE `character_talent`
+CHANGE `specMask` `talentGroup` tinyint(3) unsigned NOT NULL DEFAULT '0';
```

Comment:
specMask cannot be converted to talentGroup since it is a totally different field of TC.

AzerothCore is using a mask, developed by SunwellCore, where:

1 -> spec 1
2 -> spec 2
3 -> spec 1 and 2

TrinityCore instead use 0 for first spec, and 1 for second spec...if a talent is present on both spec, it is present twice

specMask is also used on character_spell

I think they cannot be never synced with TC